packages/vm/test/t8n/helpers.ts

```diff
@@ -84,7 +89,7 @@ export function getArguments() {
     reward: BigInt((<any>args)['state.reward']),
     chainid: BigInt((<any>args)['state.chainid']),
   }
-
+  args.trace = (<any>args)['trace']
```

Comment:
Note for myself: I should update these types, not sure why the types are not cast or set. Should do something similar as in the client where the CLI arguments are correctly typed (not this PR)